src/EnergyPlus/SetPointManager.cc

```diff
@@ -4686,8 +4693,10 @@ namespace SetPointManager {
 		Real64 SetTempAtOutHigh;
 		int SchedPtr;
 		Real64 SetPt;
+		int CtrlType;
```

Comment:
And if you want to clean this up further, this function doesn't really need local variables for SchedPtr and SetPt, it should just use this->SchedPtr and this-> SetPt directly.